examples/analytics-notification.amp.html

```diff
@@ -63,14 +63,14 @@
 </head>
 <body>
 
-  <amp-user-notification
-      layout=nodisplay
-      id="amp-user-notification1"
-      data-show-if-href="https://example.com/api/show?timestamp=TIMESTAMP"
-      data-dismiss-href="https://example.com/api/echo/post">
-      This site uses cookies to personalize content.
-     <a class="btn" on="tap:amp-user-notification1.dismiss">I accept</a>
-  </amp-user-notification>
+    <amp-user-notification
+        layout=nodisplay
+        id="amp-user-notification6"
+        data-persist-dismissal="false">
+        This notification should ALWAYS show on every page visit.
+      <a href="#learn-more">Learn more.</a>
+      <button on="tap:amp-user-notification6.dismiss">Dismiss</button>
+    </amp-user-notification>
 
   <amp-analytics data-consent-notification-id="amp-user-notification1">
```

Comment:
Change this id to match the id of the `amp-user-notification`

Reply:
I changed the id back to the previous id